apps/modernization-ui/src/apps/patient/add/contactFields/ContactFields.tsx

```diff
@@ -132,6 +132,31 @@ export default function ContactFields({ id, title }: Props) {
                         />
                     </Grid>
                 </Grid>
+                <Grid col={6}>
+                    <Controller
+                        control={control}
+                        name={`emailAddress`}
+                        rules={{
+                            pattern: {
+                                value: /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/,
+                                message: 'Please enter a valid email address (example: youremail@website.com)'
+                            },
```

Comment:
issue: There is a function to create a [standard email validation rule](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/validation/entry/validEmailRule.ts), might want to try that to keep things consistent.